roles/common/tasks/python.yml

```diff
@@ -70,6 +70,14 @@
     until: result|succeeded
     retries: 5
 
+  - name: install shade bits for ansible modules
```

Comment:
We cannot install shade bit at system level. Above task install the shade bits in base venv.